src/main/groovy/org/fundacionjala/gradle/plugins/enforce/tasks/salesforce/deployment/Upload.groovy

```diff
@@ -42,22 +36,17 @@ class Upload extends Deployment {
      */
     @Override
     void runTask() {
-        pathUpload = Paths.get(buildFolderPath, Constants.DIR_UPLOAD_FOLDER).toString()
-        uploadPackagePath = Paths.get(pathUpload, PACKAGE_NAME).toString()
+        setup()
         createDeploymentDirectory(pathUpload)
         loadFilesChangedToUpload()
-        loadParameter()
-        loadAllFiles()
-        if (specificFilesToUpload.empty && !Util.isValidProperty(parameters, EXCLUDES) && all == Constants.FALSE) {
-            logger.warn("${Constants.ALL_FILES_UPLOAD}${projectPath}")
-            option = System.console().readLine(Constants.QUESTION_CONTINUE)
-
-        }
+        loadParameters()
+        showWarningMessage()
         if (option == Constants.YES_OPTION) {
-            loadFiles()
+            loadFilesToUpload()
             copyFilesToUpload()
             createPackage()
-            truncate(pathUpload)
+            combinePackageToUpdate(uploadPackagePath)
+            truncate()
```

Comment:
maybe we can change the truncate name by applyInterceptors or something like that.